components/compas-scl-type-list/src/compas-scl-type-list.ts

```diff
@@ -1,59 +1,47 @@
-import {
-  customElement,
-  html,
-  LitElement,
-  property,
-  TemplateResult,
-} from 'lit-element';
-import { translate } from 'lit-translate';
-import '@material/mwc-list';
-import '@material/mwc-list/mwc-list-item';
-import './compas-loading.js';
+import { html, LitElement, property, TemplateResult } from "lit-element";
+import "@material/mwc-list";
+import "@material/mwc-list/mwc-list-item";
+import "@com-pas/compas-loading";
 
 /* Event that will be used when a SCL Type is selected from a list of types. */
 export interface TypeSelectedDetail {
   type: string;
 }
 export type TypeSelectedEvent = CustomEvent<TypeSelectedDetail>;
 export function newTypeSelectedEvent(type: string): TypeSelectedEvent {
-  return new CustomEvent<TypeSelectedDetail>('typeSelected', {
+  return new CustomEvent<TypeSelectedDetail>("typeSelected", {
     bubbles: true,
     composed: true,
     detail: { type },
   });
 }
 
-@customElement('compas-scltype-list')
 export class CompasSclTypeList extends LitElement {
```

Comment:
missing JSDoc here and for every prop like how it's done in `oscd-components` 
https://github.com/openscd/oscd-components/blob/main/components/oscd-action-icon/src/oscd-action-icon.ts

Reply:
Beyond the scope of this issue. I will make a new issue out of this. Since we agreed on doing this type of documentation after docops was created

Reply:
https://github.com/com-pas/compas-oscd-docops/issues/15